### PR TITLE
Update CSS properties

### PR DESCRIPTION
**Pull request by torusrxxx**
Fix incorrect CSS property names for `border-top-left-radius`, `border-top-right-radius`, `border-bottom-left-radius`, `border-bottom-right-radius`

### CODE REVIEW

File: src/freenet/client/filter/CSSTokenizerFilter.java [hunk 1]
```diff
@@ -741,27 +741,12 @@ else if("border".equalsIgnoreCase(element))
 		else if("border-radius".equalsIgnoreCase(element))
 		{
 			auxilaryVerifiers[65]=new CSSPropertyVerifier(Arrays.asList("/"),null,null,null,true);
-			elementVerifiers.put(element,new CSSPropertyVerifier(null,ElementInfo.VISUALMEDIA,null,Arrays.asList("64<1,4>", "64<1,4> 65 64<1,4>")));
+			elementVerifiers.put(element,new CSSPropertyVerifier(null,ElementInfo.VISUALMEDIA,null,Arrays.asList("40<1,4>", "40<1,4> 65 40<1,4>")));
```
Comment (ArneBab) [inline src/freenet/client/filter/CSSTokenizerFilter.java:744]: [40] is only added conditionally for padding (see line 1353), 64 always exists.

Comment (torusrxxx) [reply src/freenet/client/filter/CSSTokenizerFilter.java:744]: [40] always exists since c5ddfc86935432cc01e0daf35bba13e4fd752a3c

Comment (ArneBab) [reply src/freenet/client/filter/CSSTokenizerFilter.java:744]: Ok — I missed that. Thanks!

**Comment by ArneBab** [comment]
Merged — thank you!

**Pull request merged**